main.c

```diff
@@ -118,6 +119,17 @@ main(int argc, char *argv[])
 				exit(1);
 			}
 			break;
+		GETOPT_OPT("--passphrase-environ"):
+			if ((passwd_environ = getenv("SCRYPT_PASSPHRASE"))
```

Comment:
On second thought, we might as well make this `GETOPT_OPTARG` and read from `optarg` here.